data/technique_promises.json

```diff
@@ -6254,6 +6298,14 @@
                     "tool_available",
                     "tool_delivery"
                 ]
+            },
+            "T1547.014": {
```

Comment:
consider adding "tool_available" and "tool_delivery" to the requires section for T1547.014. From https://attack.mitre.org/techniques/T1547/014/ : 

"Adversaries can abuse these components to execute malware, such as remote access tools, to maintain persistence through system reboots."